03-interpolated-defaults.tf

```
@@ -1,10 +1,31 @@
 locals {
 
   sub_env        = var.environment == "ptlsbox" ? "ptl-sbox" : var.environment
-  aad_group_name = contains(["ptlsbox", "ptl"], var.environment) ? "DTS Contributors (sub:dts-sharedservices${local.sub_env})" : "DTS Contributors (sub:dts-sharedservices-${local.sub_env})"
-
   
-  project = length(regexall(".*sds.*", lower(var.developers_group))) > 0 ? "sds" : "cft"
+  aad_group_name = contains(["ptlsbox", "ptl"], var.environment) ? keys(local.contributors_group["${local.business_area}-${var.environment}"])[0] : keys(local.contributors_group["${local.business_area}-env"])[0]
+
+  contributors_group = {
+    "cft-ptl" = {
+      "DTS Contributors (sub:dts-cftptl-intsvc)" = {}
+    }
+    "cft-ptlsbox" = {
+      "DTS Contributors (sub:dts-cftsbox-intsvc)" = {}
+    }
+    "cft-env" = {
+      "DTS Contributors (sub:dcd-cftapps-${local.sub_env})" = {}
+    }
+    "sds-ptl" = {
+      "DTS Contributors (sub:dts-sharedservicesptl)" = {}
+    }
+    "sds-ptlsbox" = {
+      "DTS Contributors (sub:dts-sharedservicesptl-sbox)" = {}
+    }
+    "sds-env" = {
+      "DTS Contributors (sub:dts-sharedservices-${local.sub_env})" = {}
+    }
+  }
+
+  business_area = var.business_area == "cft" ? var.business_area : "sds"
```

we generally pass through sds instead of cross cutting I think so I don't think you would need this

---

Seems we're passing cross-cutting in [aks-sds-deploy](https://dev.azure.com/hmcts/PlatformOperations/_build/results?buildId=388970&view=logs&j=fbe5e610-086d-53cd-fb9a-f5a3d2cb4fdf&t=aa697d24-2372-5379-79e2-23425abbeabc&l=405)

---

ok in that case I would suggest overriding the specific one we know about and not special casing cft as that doesn't seem right
```suggestion
  business_area = var.business_area == "cross-cutting" ? "sds : var.business_area
```